src/tracedpromise.js

```diff
@@ -54,20 +54,26 @@ export default class TracedPromise {
     /**
      * Constructs anew TracedPromise
      *
-     * @param {Span} parent - the parent of the span created by this promise.
-     *        Pass `null` for a promise that does not have a parent.
+     * @param {Object} options - the options used to create the span for this
```

Comment:
Can you clarify in the docs that this matches the options available in `startSpan`?